HelpMyStreetFE/HelpMyStreetFE/Services/Requests/FilterService.cs

```diff
@@ -271,29 +265,15 @@ private SortAndFilterSet GetOpenRequestsNotMatchingCriteriaDefaultSortAndFilterS
                     .Insert(0, new FilterField<SupportActivities>() { Value = SupportActivities.CommunityConnector, IsSelected = true });
             }
 
-            if (user.SupportActivities.Intersect(_requestSettings.Value.NationalSupportActivities).Count() > 0)
+            filterSet.MaxDistanceInMiles = new List<FilterField<int>>
             {
-                filterSet.MaxDistanceInMiles = new List<FilterField<int>>
-                    {
-                        new FilterField<int> { Value = 0, Label = "My street only" },
-                        new FilterField<int> { Value = 1, Label = "Within 1 mile" },
-                        new FilterField<int> { Value = 5, Label = "Within 5 miles" },
-                        new FilterField<int> { Value = 10, Label = "Within 10 miles" },
-                        new FilterField<int> { Value = 20, Label = "Within 20 miles" },
-                        new FilterField<int> { Value = 999, Label = "Show all", IsSelected = true },
-                    };
-            }
-            else
-            {
-                filterSet.MaxDistanceInMiles = new List<FilterField<int>>
-                    {
-                        new FilterField<int> { Value = 0, Label = "My street only" },
-                        new FilterField<int> { Value = 1, Label = "Within 1 mile" },
-                        new FilterField<int> { Value = 5, Label = "Within 5 miles" },
-                        new FilterField<int> { Value = 10, Label = "Within 10 miles" },
-                        new FilterField<int> { Value = 20, Label = "Within 20 miles", IsSelected = true },
-                    };
-            }
+                new FilterField<int> { Value = 0, Label = "My street only" },
+                new FilterField<int> { Value = 1, Label = "Within 1 mile" },
+                new FilterField<int> { Value = 5, Label = "Within 5 miles" },
+                new FilterField<int> { Value = 10, Label = "Within 10 miles" },
+                new FilterField<int> { Value = 20, Label = "Within 20 miles" },
```

Comment:
We should check with Laura whether she would like to keep the "20 miles" option as well as "Show all", or not.  For most users, the two values will be equivalent.